src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -7,6 +7,9 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
+    <!-- CSC adds typeforwards for several internal nested types to the reference assembly, however
```

Comment:
We should probably fix this and avoid the big hammer of ignoring missing types as it potentially causes us to have a whole. Can you please file an issue for this.